scripts/linters/css_linter_test.py

```diff
@@ -93,35 +93,30 @@ def mock_communicate():
             subprocess, 'Popen', mock_popen)
 
         third_party_linter = css_linter.ThirdPartyCSSLintChecksManager(
-            CONFIG_PATH, [VALID_CSS_FILEPATH], True)
+            CONFIG_PATH, [VALID_CSS_FILEPATH])
         with self.print_swap, popen_swap, self.assertRaisesRegexp(
             SystemExit, '1'):
             third_party_linter.perform_all_lint_checks()
 
     def test_perform_all_lint_checks_with_no_files(self):
         third_party_linter = css_linter.ThirdPartyCSSLintChecksManager(
-            CONFIG_PATH, [], False)
-        with self.print_swap:
-            third_party_linter.perform_all_lint_checks()
-        self.assert_same_list_elements(
-            ['There are no HTML or CSS files to lint.'],
-            self.linter_stdout)
-        self.assert_failed_messages_count(self.linter_stdout, 0)
+            CONFIG_PATH, [])
+        lint_task_report = third_party_linter.perform_all_lint_checks()
+        self.assertEqual(
+            'There are no HTML or CSS files to lint.',
+            lint_task_report[0].all_messages[0])
+        self.assertEqual('CSS lint', lint_task_report[0].name)
+        self.assertFalse(lint_task_report[0].failed)
 
     def test_perform_all_lint_checks_with_valid_file(self):
         third_party_linter = css_linter.ThirdPartyCSSLintChecksManager(
```

Comment:
Instead of running the third_party checks, we should much subprocces and assert the subprocces was called with a given command. [The moto of this file is to have  unit test for the linter script.]

This needs a bit of audit, I would suggest creating an issue and leave it as later work!

Reply:
Done.